dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/findExtensionLenient.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.gradle.internal
+
+import org.gradle.api.Project
+
+/**
+ * Try and get an extension from [Project.getExtensions], or `null` if it's not present.
+ *
+ * Handles as many errors as possible.
```

Comment:
Nit: This sounds too vague, doesn't it? I'd suggest either removing it or making it more specific